src/inmanta/agent/config.py

```diff
@@ -116,8 +116,11 @@
     "config",
     "agent-repair-interval",
     600,
-    "The number of seconds between two repair runs (full deploy) of the agent or a cron expression of when to run the repair run"
-    + "Set this to 0 to disable the scheduled repair runs.",
+    "Either the number of seconds between two repair runs (full deploy) of the agent or a cron-like expression"
+    "If a cron-like expression is specified, a repair will be run following a cron-like time-to-run specification, interpreted in UTC"
+    "(e.g. `min hour dom month dow`). A repair will be requested at the scheduled time. Note that if a cron expression is use"
+    "the 'agent_repair_splay_time' setting will be ignored"
+    "Setting this to 0 to disable the scheduled repair runs.",
```

Comment:
This whole block is missing spaces, I hadn't noticed before.

Reply:
indeed, I added them